### PR TITLE
[DOCS] added validation split while training in example

### PR DESCRIPTION
**Pull request by aneesh-joshi**
As per the issue: https://github.com/keras-team/keras/issues/8796
This adds a validation split while training.
Other things are kept unchanged

### CODE REVIEW

**Comment by fchollet** [comment]
Thank you for the PR. Both ways are valid, but show different functionality.

The purpose of the examples is just to demonstrate Keras functionality. Meanwhile, the purpose of using a separate validation set is to avoid "validation set overfitting", i.e. avoid making model-tuning choices that leak validation set information into the final model. This is not a problem with this example since the architecture choices are arbitrary, rather than tuned for performance on the test set.

Thus, we won't merge this change.

**Comment by aneesh-joshi** [comment]
@fchollet 
Ok.
Thank you for considering.
